code/game/machinery/kitchen/microwave.dm

```diff
@@ -341,6 +348,31 @@
 
 	var/datum/recipe/recipe = select_recipe(available_recipes,src)
 	var/obj/cooked
+
+	for(var/obj/O in contents)
+		if(istype(O,/obj/item/weapon/kitchen/utensil) && !(O.melt_temperature == MELTPOINT_PLASTIC))
+			if (!wzhzhzh(4))
+				abort()
+				return
+			wzhzhzh(4)
```

Comment:
If this gets reached, wzhzhzh already got called successfully in the if statement. Is it intentional to call this twice?

Reply:
fixed